.github/workflows/ci.yaml

```diff
@@ -92,11 +79,8 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
 
-      - name: get-job-time
-        run: curl --user tigunov-it:${{ secrets.GITHUB_TOKEN }} https://api.github.com/repos/tigunov-it/angular-react-starter-test/actions/runs/${GITHUB_RUN_ID}/timing  > time.txt
-
       - name: Parse job time
-        run: echo "::set-output name=JOB_TIME::$(cat time.txt | grep "duration_ms")"
+        run: echo "::set-output name=JOB_TIME::$(uptime)"
```

Comment:
so we need to know the exact value in seconds. Like the duration was 72seconds. Which is displayed in the final email.

Reply:
I use "date +%s" at the beginning and at the end workflow.
It works correctly